src/i_input.c

```diff
@@ -636,6 +692,67 @@ static int TranslateKey(int scancode)
     }
 }
 
+
+// Get the localized version of the key press. This takes into account the
+// keyboard layout, but does not apply any changes due to modifiers, (eg.
+// shift-, alt-, etc.)
+
+static int GetLocalizedKey(SDL_Keysym *sym)
+{
+    // When using Vanilla mapping, we just base everything off the scancode
+    // and always pretend the user is using a US layout keyboard.
+    if (vanilla_keyboard_mapping)
+    {
+        return TranslateKey(sym);
+    }
+    else
+    {
+        int result = sym->sym;
+
+        if (result < 0 || result >= 128)
+        {
+            result = 0;
+        }
+
+        return result;
+    }
+}
+
+static int GetTypedChar(SDL_Keysym *sym)
+{
+    int result = TranslateKey(sym);
```

Comment:
Shouldn't this rather be `GetLocalizedKey()`?

Reply:
It's from Choco: https://github.com/chocolate-doom/chocolate-doom/blob/bb9bb2324852db731d81ee8c1f16a9b0b0fc31df/src/i_input.c#L176